bridgescaler/distributed.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from copy import copy
+
+
+class DStandardScaler(object):
+    """
```

Comment:
Can you add a test for the distributed scaler?

Reply:
Done! Found a bunch of issues in the process that I've fixed.